bundle/Resources/translations/ibexa_content_url.en.yaml

```diff
@@ -0,0 +1,5 @@
+tab.urls.siteaccess.headline.content_tree: 'Content tree URLs (inside configured Siteaccess Content tree)'
+tab.urls.siteaccess.headline.external: 'External URLs (outside configured Siteaccess Content tree)'
```

Comment:
Rename to `Siteaccess URLs outside of the configured Content tree root`.

Reply:
fixed in https://github.com/netgen/ibexa-admin-ui-extra/pull/2/commits/b70c2e558415c9e07aa8afefed34860e84a49905